src/app/page.tsx

```diff
@@ -208,8 +220,10 @@ export default function HomePage() {
         >
           <div className="bg-black bg-opacity-70 p-16">
             <div className="flex flex-col items-center justify-center text-white lg:mb-48 lg:ml-64 lg:mr-64 lg:mt-32">
-              <h1 className="p-16">OUR STORY</h1>
-              <p className="text-xl">
+              <h1 className="p-16" style={{ textAlign: "center" }}>
```

Comment:
adding `text-center` to the parent div should save us having to add the styling manually to both paragraphs